openmls/src/extensions/mod.rs

```diff
@@ -22,25 +22,28 @@
 //! - [`RequiredCapabilitiesExtension`] (GroupContext extension)
 
 use openmls_traits::crypto::OpenMlsCrypto;
-pub(crate) use serde::{Deserialize, Serialize};
+use serde::{Deserialize, Serialize};
 use std::{
     convert::TryFrom,
     fmt::Debug,
     io::{Read, Write},
 };
 use tls_codec::*;
 
+// Private
 mod capabilities_extension;
-mod errors;
 mod external_key_id_extension;
 mod life_time_extension;
 mod parent_hash_extension;
 mod ratchet_tree_extension;
 mod required_capabilities;
+use errors::*;
```

Comment:
Should this line not be above the `// Private` line?

Reply:
I put the `use` of submodules in the private section. the other `use` statements are for things that are no submodules.